Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

EBOOKCharsetConverter.cpp:127:7: error: use of undeclared identifier 'TRUE'
      TRUE, TRUE, &status)
      ^
EBOOKCharsetConverter.cpp:127:13: error: use of undeclared identifier 'TRUE'
      TRUE, TRUE, &status)
            ^

--- a/src/lib/EBOOKCharsetConverter.cpp.orig	2018-01-01 11:16:28 UTC
+++ b/src/lib/EBOOKCharsetConverter.cpp
@@ -124,7 +124,7 @@ bool EBOOKCharsetConverter::convertBytes(const char *c
       m_converterToUTF8.get(), m_converterToUnicode.get(),
       &outText, outText + out.size(), &inText, inText + length,
       nullptr, nullptr, nullptr, nullptr,
-      TRUE, TRUE, &status)
+      true, true, &status)
     ;
     if (status==U_BUFFER_OVERFLOW_ERROR)
     {
